lib/tasks/reset_permissions.rake

```diff
@@ -22,6 +22,7 @@ namespace :permissions do
       user.password = password
       if user.save
         puts "Reset to user: #{user.login}, password: #{password}"
+        puts "WARNING: You need to change the password in foreman.yml manually!"
```

Comment:
Looks better, but I'd rephrase it a bit. What about:
```suggestion
        puts "WARNING: You need to change the password for hammer in ~/.#{user.login}/.hammer/cli.modules.d/foreman.yml manually!"
```
?

Reply:
What if Foreman user doesn't have same name as in Linux account?

Reply:
True, updated.

Reply:
So how does this version reflect the root location? From what @stejskalleos told me, there is more than one location where the yaml file can be present. Should there be all of them?

Reply:
Well, if the user is root then `~/` gets translated into the root directory, doesn't it?

There are few places, yes. The main one is in `/etc/hammer`, but there should live settings which can be re-used by different users. Per user settings (such as their passwords) should live in their home `~/.hammer` directories.

Although I'm not sure if we force those directories, it's recommended.

Reply:
I forgot about that. Thank you. I will implement it according to your suggestion.